src/sentry/api/endpoints/codeowners/details.py

```diff
@@ -63,7 +72,7 @@ def put(self, request: Request, project, codeowners) -> Response:
 
             analytics.record(
                 "codeowners.updated",
-                user_id=request.user.id if request.user and request.user.id else None,
+                user_id=getattr(request.user, "id", None),
```

Comment:
This is a very small nit, but technically this change could return something different in an edge case. If `request.user.id == 0` or something else falsey, this will return `0` instead of `None`. That may be fine, I just wanted to point it out.

Reply:
If you still want to shorten this you can do `getattr(request.user, "id", None) or None`